pagopa/pn-frontend
# new notification api migration - pn-10941

## Short description
Migrated new notification and preload document api to bff ones

## List of changes proposed in this pull request
- Auto generated clients
- Removed custom api client
- Updated redux layer
- Updated tests

## How to test
Login with PA -> create a notification -> check that documents are uploaded correctly going to the detail and downloading them
